Sources/BrowserServicesKit/SecureVault/SecureVaultModels.swift

```diff
@@ -197,20 +226,72 @@ public struct SecureVaultModels {
 
     public struct Identity {
 
+        private static let mediumPersonNameComponentsFormatter: PersonNameComponentsFormatter = {
+            let nameFormatter = PersonNameComponentsFormatter()
+            nameFormatter.style = .medium
+            return nameFormatter
+        }()
+        
+        private static let longPersonNameComponentsFormatter: PersonNameComponentsFormatter = {
+            let nameFormatter = PersonNameComponentsFormatter()
+            nameFormatter.style = .long
+            return nameFormatter
+        }()
+        
+        private var nameComponents: PersonNameComponents {
+            var nameComponents = PersonNameComponents()
+
+            nameComponents.givenName = firstName
+            nameComponents.middleName = middleName
+            nameComponents.familyName = lastName
+
+            return nameComponents
+        }
+        
+        public var formattedName: String {
+            return Self.mediumPersonNameComponentsFormatter.string(from: nameComponents)
+        }
+        
+        public var longFormattedName: String {
+            return Self.longPersonNameComponentsFormatter.string(from: nameComponents)
+        }
+        
+        var autofillEqualityName: String?
+        var autofillEqualityAddressStreet: String?
```

Comment:
Need to improve this, it's not great since initializers in other files are having to ensure that they set these values too, making it hard to write a new initializer and know that you're doing it correctly.

Reply:
Right, making these vars lazy or turning them into functions backed by a private var that is initialized on demand makes things even more complicated, because `SecureVaultAutofillEquatable` needs to define its function `mutable` and taking `inout` parameter.

There is probably no nice solution for handling it behind the scenes.